src/main/java/duke/util/Ui.java

```diff
@@ -0,0 +1,129 @@
+            System.out.println(String.format("%sHere is your list", padding));
+            int i = 1;
+            for (Task task : storedItems) {
+                System.out.println(String.format("%s%d. %s", padding, i, task));
+                i++;
+            }
+        }
+
+        System.out.println(String.format("%s%s", padding, uselessLine));
+    }
+
+    /**
+     * Print the status after successfully storing a task
+     * @param task
+     * @param storedItems
+     */
+    public static void storeUserInput(Task task, ArrayList<Task> storedItems) {
+        System.out.println(String.format("%s%s", padding, uselessLine));
+        System.out.println(String.format("%s%s%s", padding, addedPhrase, task));
+        System.out.println(String.format("%sNow you have %d tasks", padding, storedItems.size()));
+        System.out.println(String.format("%s%s", padding, uselessLine));
+    }
+
+    /**
+     * Print the status after successfully marking a task as Done
+     * @param pos
+     * @param storedItems
+     */
+    public static void markItemAsDone(int pos, ArrayList<Task> storedItems) {
+        System.out.println(String.format("%s%s", padding, uselessLine));
+        System.out.println(String.format("%sNice nice. I've marked the task as done for you", padding));
+        System.out.println(String.format("%s   %s", padding, storedItems.get(pos)));
+        System.out.println(String.format("%s%s", padding, uselessLine));
+    }
+
+    /**
+     * Print the status after successfully deleting a task
+     * @param t
+     * @param storedItems
+     */
+    public static void deleteItem(Task t, ArrayList<Task> storedItems) {
+        System.out.println(String.format("%s%s", padding, uselessLine));
+        System.out.println(String.format("%sI've removed this task for you", padding));
+        System.out.println(String.format("%s   %s", padding, t));
+        System.out.println(String.format("%sYou have %d tasks left", padding, storedItems.size()));
+        System.out.println(String.format("%s%s", padding, uselessLine));
+    }
+
+    /**
+     * React to a blank user input
+     */
+    public static void blankInput() {
+        System.out.println(String.format("%s%s", padding, uselessLine));
+        System.out.println(String.format("%sPlease type something. Don't leave it blank, plsss!", padding));
+        System.out.println(String.format("%s%s", padding, uselessLine));
+    }
+
+    /**
+     * Print the resulting foundList of a find request
+     * @param foundList
+     */
+    public static void printFoundList(ArrayList<Task> foundList) {
+        System.out.println(String.format("%s%s", padding, uselessLine));
+        System.out.println(String.format("%sHere are the matching tasks in your list:", padding));
+        if (foundList.isEmpty()) {
+            System.out.println(String.format("%s~~~~~Oops! There are no matching tasks!", padding));
+        } else {
+            int i = 1;
+            for (Task task : foundList)
```

Comment:
I think you need to add a curly brace even if the for loop only covers one line of code.